### PR TITLE
LPS-177072 TR2 Automated Test | master

### PR DESCRIPTION
**Pull request by joelrocha68**
[LPS-177073](https://issues.liferay.com/browse/LPS-177073)
[LPS-177072](https://issues.liferay.com/browse/LPS-177072)

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:site-initializer-testray:nocompile

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase [hunk 1]
```diff
@@ -36,6 +44,54 @@ definition {
...
+			Type(
+				key_fieldLabel = "Name",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "userfn userln");
+
+			Button.click(button = "Apply");
+		}
+
+		task ("Then can see the filters apply in the list by user name") {
+			Testray.assertItemName(key_name = "userfn userln");
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase:89]: You can use AssertElementPresent here

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase [hunk 2]
```diff
@@ -36,6 +44,54 @@ definition {
...
+	@priority = 4
+	test AdminCanFilterForNonexistantManageUsers {
+		task ("When the user filters a user by its non-existent name") {
+			TestrayLayout.clickManageOnSidebar();
+
+			TestrayManage.selectOptionsManage(itemValue = "Manage Users");
+
+			TestrayFilter.clickFilter();
+
+			Type(
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase:57]: Please put this in a macro?

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase [hunk 3]
```diff
@@ -36,6 +44,54 @@ definition {
 		}
 	}
 
+	@description = "Story ID: LPS-173531 | Test Summary: Verify if as an Administrator I should not see manage users that don't exist when I try filtering for them"
+	@priority = 4
+	test AdminCanFilterForNonexistantManageUsers {
```
[the remark this answers is not in the document]
Comment (joelrocha68) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase]: Done

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase [hunk 4]
```diff
@@ -36,6 +44,54 @@ definition {
...
+		}
+
+		task ("Then can see message no results found") {
+			TestrayFilter.assertNoResultsFound();
+		}
+	}
+
+	@description = "Story ID: LPS-173531 | Test Summary: Verify if as an Administrator I should be able to filter manage users by name"
+	@priority = 5
+	test AdminCanFilterManageUsersByName {
```
[the remark this answers is not in the document]
Comment (joelrocha68) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase]: Done

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase (hunk 1 above)
Comment (joelrocha68) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayManageUsers.testcase]: Done

**Comment by joelrocha68** [comment]
ci:test:site-initializer-testray:nocompile

**Comment by joelrocha68** [comment]
ci:forward

**Pull request closed without merging**